src/main/java/io/github/opencubicchunks/cubicchunks/core/CubicChunksConfig.java

```diff
@@ -132,6 +132,10 @@
     @Config.Comment("Vanilla compatibility generator type, which will convert vanilla world type generators output in cubic")
     public static String compatibilityGeneratorType = "cubicchunks:default";
 
+    @Config.LangKey("cubicchunks.config.storage_format")
+    @Config.Comment("Storage format type. Warning: changing this WILL break existing worlds!")
+    public static String storageFormat = "cubicchunks:anvil3d";
```

Comment:
That's not how this config option should work. It should work as a value to be used for new world, without breaking existing worlds.

Reply:
actually, the comment is wrong - it only affects new worlds, but *doesn't* break existing worlds (i changed that while working on it). will update comment accordingly.